### ISSUE TITLE
mkmf not included in installation

### ISSUE THREAD

**Issue by uramirez8707** [NOAA-GFDL/fre-cli#229]
**Describe the bug**
mkmf is not included in the installation,  fre make will fail when trying to use it

**To Reproduce**
Install fre-cli and attempt to run fre make, it will fail because it cannot find mkmf in the path

**Expected behavior**
fre-cli should include mkmf (i.e all of its dependencies) 

**Additional context**
You can work around this by cloning mkmf and adding it to your path, or loading fre/bronx-22 if on gaea.

**Comment by singhd789** [comment]
@uramirez8707 Just fyi, the module for fre/bronx should already be included in the platforms.yaml, as shown  here (this needs to be updated) for gaea: 
https://github.com/NOAA-GFDL/fre-cli/blob/12b35f0059141afbe2c295b5d0508305561646c6/fre/make/tests/AM5_example/yaml_include/platforms.yaml#L5

**Comment by ceblanton** [comment]
If mkmf were conda-installable it could be included as a proper dependency. This would be the most direct solution IMO.

**Comment by singhd789** [comment]
Yes, if someone could make it a conda package, that might be easier

**Comment by ilaflott** [comment]
[export PATH=$PATH:$PWD ?](https://github.com/NOAA-GFDL/fre-cli/blob/9fa8b27e326fa51b05765059654bd157d03fcbc1/meta.yaml#L12)

**Comment by singhd789** [comment]
Is this not adding it though?

https://github.com/NOAA-GFDL/fre-cli/blob/9fa8b27e326fa51b05765059654bd157d03fcbc1/.github/workflows/publish_conda.yml#L18C11-L18C45

**Comment by singhd789** [comment]
Related to @ilaflott PR: https://github.com/NOAA-GFDL/mkmf/pull/75

**Comment by ilaflott** [comment]
this will be closed by #512
